tests/test_compile.py

```diff
@@ -20,18 +20,22 @@
 import os
 import sys
 from kapitan.cli import main
-from kapitan.utils import get_directory_hash
+from kapitan.utils import directory_hash
 
 
 class CompileTest(unittest.TestCase):
     def setUp(self):
         os.chdir(os.getcwd() + '/examples/kubernetes/')
 
     def test_compile(self):
-        sys.argv = ["kapitan", "compile"]
+        sys.argv = ["kapitan", "compile", "-f"]
         main()
-        compiled_dir_hash = get_directory_hash(os.getcwd() + '/compiled')
-        test_compiled_dir_hash = get_directory_hash(os.getcwd() + '/../../tests/test_kubernetes_compiled')
+        try:
+            os.remove(os.getcwd() + '/compiled/.kapitan_cache')
```

Comment:
I think instead of ignoring `/compiled/.kapitan_cache`, we should actually add it to the `tests/test_kubernetes_compiled` folder.

This way we can test the cache file is being generated correctly